docs/modules/ROOT/pages/how-to/fine-grained-access-examples.adoc

```diff
@@ -0,0 +1,109 @@
+= Fine grained access control examples
+
+{product} allows fine-grained control over which projects a user may access through standard RBAC roles.
+
+This is useful for organizations that have multiple projects and want to restrict some users access to only certain projects.
+
+You can either create https://portal.appuio.cloud/teams[Teams], which show up as an OpenShift group `organization+team` or assign users directly.
+
+The examples below assume the following structure:
+
+[source]
+----
+mycompany (organization)
+├── mycompany-user1 (user)
+├── developers (team)
+└── operations (team)
+----
+
+== Prerequisites for all examples
+
+For this guide, it's assumed that:
+
+* You are logged in to {product} using the `oc login` command.
+* You have installed `kubectl`.
```

Comment:
```suggestion
* You have installed `kubectl`.
```

Is there a technical reason to use `kubectl`? If not, I suggest to use `oc` exclusively on this page. Or mention that `oc` and `kubectl` can (mostly) be used interchangeable.

Reply:
there's none. i switch the tutorial to `oc` only.